### ISSUE TITLE
Trajectory drawback colour change

### ISSUE THREAD

**Issue by TobSick** [ImpactDevelopment/ImpactIssues#2029]
**Describe the solution you'd like**
When drawing back a bow, could there be an option for the trajectory's colour to change upon different drawback strengths?

**Additional context**
This is similar to some resource packs which change the colour of the bow string on different drawback strengths.

**Comment by S-B99** [comment]
This is dependent on the resource pack you're using, you can't change that color without inserting a custom texture which is difficult because people can be using custom texturepacks.

**Comment by TobSick** [comment]
I'm talking about the Trajectory line changing colour, not the bow, that was just an example. Surely that wouldn't require a custom resource pack?

**Comment by S-B99** [comment]
It would, as the color is done by the resourcepack.

**Comment by TobSick** [comment]
What? But the colour on tracer lines are able to change colour depending on distance. Is the trajectory line different?

**Comment by S-B99** [comment]
Ohhh, I misread the issue. I thought you meant the drawback line on bows, now the drawn trajectory that impact adds.

**Comment by TobSick** [comment]
Yeah sure. You're just trying to shutdown every comment I have on Kami and Impact. Hmm, this conspiracy goes deep.

**Comment by S-B99** [comment]
What? I closed your issues in Blue and gave an explanation as to why, I closed this here by mistake as I misunderstood your issue.
